recipes/CodeLLM_DevKit/generate_unit_tests.ipynb

```diff
@@ -245,7 +215,7 @@
     "                        \n",
     "                        # Print the instruction\n",
     "                        print(f\"Instruction:\\n{prompt}\\n\")\n",
-    "                        print(f\"Generating test case ...\\n\")\n",
+    "                        print(f\"Generating test case and it will take few minutes (or even seconds) based on where the model has been hosted...\\n\")\n",
```

Comment:
This is good, but I think we need statement earlier. It takes about 13s on my host to print out the Class ([line 178](https://github.com/ibm-granite-community/granite-code-cookbook/pull/60/files#diff-4d065a2a0b2227ca4fef0b16f386e27c402cfd66390a3067fc551cfac4305f03R178)):
> Class: org.apache.commons.cli.GnuParser

Similar with the other recipes.